src/main.rs

```diff
@@ -299,7 +431,11 @@ fn tarball(
         .arg(config.package.tarball(output))
         .arg(binary_name);
     if let Some(lic) = license {
-        command.arg(lic.path());
+        // NOTE: -C is required, as license may not be in the current directory,
+        // but we want it to end up at the root of the tarball.
+        command.arg("-C");
```

Comment:
So the original behaviour is a bug, to assume that LICENSE is at the top-level?

Reply:
It wasn't a bug per se, as in the previous version, LICENSE was only looked for in the current directory. But since the function signature accepts a `DirEntry` for `license` which may be elsewhere in disk, for completeness sake we should change to it's directory. This allows this function to be re-used in `crates`.